app/views/root/_base_page.html.erb

```diff
@@ -21,5 +21,7 @@
 </main>
 
 <% if local_assigns.fetch(:show_related_items, true) %>
-  <div id="related-items"></div>
+  <aside>
+    <div id="related-items"></div>
```

Comment:
slimmer looks for `div#related-items` for the replace. I don't fancy updating slimmer.